pkg/api/openshiftcluster.go

```diff
@@ -151,19 +151,25 @@ type ProvisioningState string
 
 // ProvisioningState constants
 const (
-	ProvisioningStateCreating      ProvisioningState = "Creating"
-	ProvisioningStateUpdating      ProvisioningState = "Updating"
-	ProvisioningStateAdminUpdating ProvisioningState = "AdminUpdating"
-	ProvisioningStateDeleting      ProvisioningState = "Deleting"
-	ProvisioningStateSucceeded     ProvisioningState = "Succeeded"
-	ProvisioningStateFailed        ProvisioningState = "Failed"
+	ProvisioningStateCreating         ProvisioningState = "Creating"
+	ProvisioningStateUpdating         ProvisioningState = "Updating"
+	ProvisioningStateAdminUpdating    ProvisioningState = "AdminUpdating"
+	ProvisioningStateOperatorUpdating ProvisioningState = "OperatorUpdating"
```

Comment:
Why not reuse the state?

Reply:
Because presently we don't parameterise -- although that does sound like a good idea